graphqlapi/src/main/java/io/stargate/graphql/web/resources/DmlResource.java

```diff
@@ -59,7 +68,7 @@ public void get(
       @Context HttpServletRequest httpRequest,
       @Suspended AsyncResponse asyncResponse) {
 
-    GraphQL graphql = getDefaultGraphql(httpRequest, asyncResponse);
+    GraphQL graphql = getGraphql(graphqlCache.getDefaultKeyspaceName(), httpRequest, asyncResponse);
```

Comment:
what will `getDefaultKeyspaceName()` return in this context?

Reply:
It will be `null` if the default keyspace feature is disabled. So I need to check and return a 404 like in the previous version.